ui/DesignSystem/Component/Remote.svelte

```diff
@@ -11,28 +10,31 @@
   export let store: Readable<remote.Data<any>>;
   export let context: string | undefined = undefined;
 
-  // Shorthand for casting these states
-  type ErrorState = { status: remote.Status.Error; error: error.Error };
+  export let disableErrorLogging: boolean = false;
+
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   type SuccessState = { status: remote.Status.Success; data: any };
 
   // If no error slot was provided, svelte will instantiate the fallback div
   let noErrorSlotProvided: HTMLDivElement;
 
-  $: if ($store.status === remote.Status.Error && !!noErrorSlotProvided) {
-    console.error("Remote error", ($store as ErrorState).error);
-    notification.error(($store as ErrorState).error.message);
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  let storeValue: remote.Data<any>;
```

Comment:
What's the `any` mean here? :thinking:

Reply:
The `any` type in TypeScript can be coerced into any other type. We are using it to work around the fact that components in svelte cannot be generic (unlike in react). So instead of having a type parameter `T` that is brought into scope by the component and used as `remote.Data<T>` we need to fall back to `any`.

Reply:
Ah, ok. Thanks for the explainer :+1: